pkg/openstack/loadbalancer.go

```diff
@@ -124,6 +126,7 @@ type serviceConfig struct {
 	lbPublicSubnetSpec          *floatingSubnetSpec
 	nodeSelectors               map[string]string
 	keepClientIP                bool
+	poolLbMethod                *v2pools.LBMethod
```

Comment:
do we really need this to be a pointer and `v2pools.LBMethod` type? Let's make it a regular string and avoid type conversion useless type conversions. We can keep only one when we define `pools.CreateOpts`.